### ISSUE TITLE
Matern versus Exponential Covariance models

### ISSUE THREAD

**Issue by chrisdelat** [GeoStat-Framework/GSTools#102]
Could you please explain why the exponential model and Matern model with nu = 0.5 produce different normalized autocorrelation functions in the attached plot? My understanding from literature is that when nu=0.5 the Matern model should simplify down to the exponential model, but I can't match this behaviour in these autocorrelation functions. They do look similar but the lag is scaled differently. 

I generated these using the following parameters:

aX = 50
anis = 10 

model_MatExp = Matern(dim=1, mean=0, var=1, len_scale=[aX, aX / anis], nu=0.5)
model_Exp = Exponential(dim=1, mean=0, var=1, len_scale=[aX, aX / anis])

Thank you!

[FigB1_compareAutoCorrFunc.pdf](https://github.com/GeoStat-Framework/GSTools/files/5180380/FigB1_compareAutoCorrFunc.pdf)

**Comment by MuellerSeb** [comment]
Hey there, you are completely right, that the Matern model coincides with the Exponential model for `nu=0.5`.
Only difference is, that the scaling, in our implementation, is different.

This could be overcome by setting the `integral_scale` in the two models:
```python
import gstools as gs

model_mat_exp = gs.Matern(dim=1, integral_scale=10, nu=0.5)
model_exp = gs.Exponential(dim=1, integral_scale=10)

ax = model_exp.plot("covariance")
model_mat_exp.plot("covariance", ax=ax)
```
![mat_exp_1](https://user-images.githubusercontent.com/19690642/92366837-f49ff980-f0f5-11ea-80e5-5cb9dd7b0cab.png)

When only setting the `len_scale` parameter as described in both implementations ([Matern](https://geostat-framework.readthedocs.io/projects/gstools/en/stable/generated/gstools.covmodel.Matern.html#gstools.covmodel.Matern) & [Exponential](https://geostat-framework.readthedocs.io/projects/gstools/en/stable/generated/gstools.covmodel.Exponential.html#gstools.covmodel.Exponential)) their integral scales differ:
```python
import gstools as gs

model_mat_exp = gs.Matern(dim=1, len_scale=10, nu=0.5)
model_exp = gs.Exponential(dim=1, len_scale=10)

print("Matern:", model_mat_exp.integral_scale)
print("Exponential:", model_exp.integral_scale)
```
Output:
```bash
Matern: 14.142135623730953
Exponential: 10
```

There are a lot of different (rescaled) implementations of the Matern model or the Exponential model (and all other models), but I was following this formula: [Wikipedia: Matern](https://en.wikipedia.org/wiki/Mat%C3%A9rn_covariance_function)
This has the advantage, that the shape-parameter `nu` will rescale the input `len_scale` (which is not the case in other implementations), so that the limit case for `nu` to infinity results in the gaussian model with a finite length scale.
One could also implement the Matern model, so that  the `len_scale` coincides with the integral scale, but I wanted to stay with the literature.

## TL;DR
The only difference in your comparison is a rescaled `len_scale` which is nothing special, since the `len_scale` parameter itself has not a consistent meaning for all models. This could be overcome by the setting the integral scale.

## Some hints for you
- The parameter `mean` can't be set in the covariance model, since it is not connected to the covariance structure of a field. You have to set it later in the `SRF` or `Krige` class.
- in GSTools, we interpret the anisotropy ratio in the inverse way: `len_scale_y = len_scale_x * anis`
- you provided two length scales but in 1D, so the second length scale will be ignored either way

**Comment by chrisdelat** [comment]
Thanks Sebastian, that's very helpful, as always. After reading up on it a bit more I figured it had to do with the dependence on `nu` in the relationship between the practical range and the scale parameter. 

Cheers.

**Comment by MuellerSeb** [comment]
Cool! :blush:
